### ISSUE TITLE
resources.Concat does not concat Javascript correctly

### ISSUE THREAD

**Issue by rluba** [gohugoio/hugo#6016]
Concatting two (or more) Javascript files can produce invalid output if one of the files ends with a comment line, eg., a sourcemap line:

# File A
```Javascript
var foo ='bar';
//# sourceMappingURL=file-a.js.map
```

# File B
```Javascript
(function () {
console.log('Hello');
})();
```
# Result of `resources.Concat`
```Javascript
var foo ='bar';
//# sourceMappingURL=file-a.js.map(function () {
console.log('Hello');
})();
```
… which changes the semantics (and, in this case, produces a syntax error).

**Comment by bep** [comment]
Pretty sure this was fixed in 35abce27cabee43cc751db55a75b927f26275833 ?

**Comment by rluba** [comment]
That’s good to hear. I used the latest release available (`v0.55.6/extended` via homebrew) and 35abce2 isn’t part of that, yet.

**Comment by rluba** [comment]
@bep When do you plan to release a version that includes this fix? Is there anything I can do to help?
